code/modules/admin/verbs/randomverbs.dm

```diff
@@ -47,6 +47,52 @@
 	admin_ticket_log(M, msg)
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Subtle Message") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/cmd_admin_mod_antag_rep(client/C in GLOB.clients, var/operation)
+	set category = "Special Verbs"
+	set name = "Modify Antagonist Reputation"
+
+	if (!holder)
+		to_chat(src, "Only administrators may use this command.")
+		return

```

Comment:
```dm
if(!check_rights(R_ADMIN))
    return
```

Reply:
The other functions right above this that are `/client/proc/cmd_admin*` use the `!holder` thing. What's the difference?

Reply:
I'm assuming it's related to the `AVerbsAdmin` list [here](https://github.com/tgstation/tgstation/blob/master/code/modules/admin/admin_verbs.dm#L43)?

Reply:
it's legacy code.

every new admin command should have an actual right assigned to it not just aholder.

Otherwise you can't give non admins holders with only certain commands.


It'll make future overhauls easier.